docs/ROADMAP.md

```diff
@@ -1,44 +1,77 @@
 # kpt roadmap
 
-Last updated: May 6th, 2022
+Last updated: May 31st, 2022
 
-Please follow the [installation](https://kpt.dev/installation/) guide for installing the latest version of kpt.
+We added new components `package orchestrator`, `Config as Data UI`,
+`Config Sync` and `Go function SDK` to the `kpt` toolchain recently. We are
+looking for feedback for all the components that will help us prioritize work.
+If you have feedback about what you think we should be working on, we encourage
+you to get in touch (e.g. by filing an issue, or using the "thumbs-up" emoji
+reaction on an issue's first comment).
 
-We are planning to build a new roadmap pending user feedback on the new package orchestration capabilities.
+## 2022
 
-## Latest releases
+---
 
-Coming soon...
+## Areas of Focus
 
-## Detailed release notes
-Please refer to the [release notes] page for more information about the latest features.
-
-## Upcoming features
+---
 
 A few areas of work are ongoing. (This is not exhaustive.)
 
 ### Package orchestration
 
-See the [package orchestration roadmap](https://github.com/GoogleContainerTools/kpt/blob/main/porch/docs/porch-roadmap.md)
+This is an area where we want to support critical scenario associated with
+day 2 package management at scale, for example supporting bulk package update,
+restoring packages to previous versions, preview package changes, admission
+control at the repository level to enforce guardrails.
+
+See the [detailed package orchestration roadmap](https://github.com/GoogleContainerTools/kpt/blob/main/porch/docs/porch-roadmap.md)
+or see [the porch issues](https://github.com/GoogleContainerTools/kpt/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Fporch)
+for more details.
+
+### Package Workflow
+
+We intend to improve workflows involving package update, constructing package
+variants, package rendering that involves generator functions, documentation
+and examples.
+See the [packaging and rendering issues](https://github.com/GoogleContainerTools/kpt/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Fhydrate%2Carea%2Fpkg%2Carea%2Fpkg-update)
+for more details.
+
+### Function Runtime
+
+We have made some progress in this area by adding support for [`podman`](https://podman.io)
+container runtime and in-cluster function evaluator for `package orchestration`
+and we we plan to explore other ways to addresss performance and dependency
+issues for running functions. Few areas that are in active considerations are:
+exploring runtimes such as `WASM` for function execution, builtin support for
+Starlark, providing common functions as built-ins in `kpt` CLI.
```

Comment:
Is the Starlark epic issue up to date?
https://github.com/GoogleContainerTools/kpt/issues/2653

Reply:
That's more of an SDK issue than a runtime issue, so I'd break it out separately  in this doc.